server/filterUpcomingScreenings.js

```diff
@@ -0,0 +1,15 @@
+export default function filterUpcomingScreenings(movieArr) {
+  return movieArr.data
+  .map((elem) => {
+    return {
+      room: elem.attributes.room,
+      date: elem.attributes.start_time,
+    };
+  })
+  .filter((elem) => {
+    if (elem.room && elem.date)
+      return Date.parse(elem.date) > Date.now();
+  })
+  .sort();
```

Comment:
The `sort()` method here probably doesn't do what you'd expect. Because the array it is being called upon is an array of objects, it won't be able to sort them in any meaningful order.